### ISSUE TITLE
README missing add Security.framework

### ISSUE THREAD

**Issue by nejj** [nxtbgthng/OAuth2Client#42]
Just followed the README but in the end after copy-pasting the following code from the read me:

``` objectivec
+ (void)initialize;
{
    [[NXOAuth2AccountStore sharedStore] setClientID:@"xXxXxXxXxXxX"
                                             secret:@"xXxXxXxXxXxX"
                                   authorizationURL:[NSURL URLWithString:@"https://...your auth URL..."]
                                           tokenURL:[NSURL URLWithString:@"https://...your token URL..."]
                                        redirectURL:[NSURL URLWithString:@"https://...your redirect URL..."]
                                     forAccountType:@"myFancyService"];
}
```

I got about 17 different "_SecTrustEvaluate" errors which interrupted the build.

Stumbled upon this http://stackoverflow.com/questions/5150982/iphone-sdk-paypal-integration-api when search for a solution - It suggested adding the Security.framework. After doing so I was able to complete the build successfully.

tl;dr Adding Security.framework to "Link Binary With Libraries" fixed above issue.

**Comment by jvjvdev** [comment]
this was driving me crazy already. thank you.
jesus, this should be on the Getting Started section.

**Comment by toto** [comment]
Thanks for this. Will add this to the README :-)

**Comment by siddyammer** [comment]
This got me as well. I spent a couple of hours trying to figure this out. It really should be on the Getting Started section.

**Comment by stigi** [comment]
@siddyammer Thanks for the ping. Looks like @toto forgot to update it 4 months ago ;)

I just added a line to the iOS getting started.
